### PR TITLE
Added option to install openHAB milestone builds

### PR DESCRIPTION
**Pull request by mstormi**
Signed-off-by: Markus Storm markus.storm@gmx.net (github: mstormi)

### CODE REVIEW

File: functions/openhab.sh [hunk 1]
```diff
@@ -2,44 +2,53 @@
...
-    echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (stable)... "
-    introtext=$introtext_stable
-    successtext=$successtext_stable
+    if [ ! -n "$TESTING" ]; then
+      echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (stable)... "
+      introtext=$introtext_stable
+      successtext=$successtext_stable
+      REPO="deb https://dl.bintray.com/openhab/apt-repo2 stable main"
+    else
+      echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (milestone)... "
```
Comment (EliasGabrielsson) [inline functions/openhab.sh]: What about:
`[openHABian] Installing or upgrading to latest openHAB milestone release (testing)` ?

File: functions/openhab.sh [hunk 2]
```diff
@@ -2,44 +2,53 @@
...
 Check the \"openHAB Release Notes\" and the official announcements to learn about additons, fixes and changes."
-  # TODO: Add back in when links are known
-  # ➡ http://www.kaikreuzer.de/2017/06/28/openhab21
-  # ➡ https://github.com/openhab/openhab-distro/releases/tag/2.1.0"
-
+  introtext_testing="You are about to install or upgrade to the latest milestone openHAB build. It contains the latest features and is supposed to run stable, but if you experience bugs or incompatibilities, please help enhancing openHAB by posting them on the community forum or by raising a Github issue.\\n
+  Please be aware that downgrading from a newer testing (milestone) build is not officially supported. Please consult with the documentation or community forum and be sure to take a full openHAB configuration backup first!"
+successtext_testing="The milestone (testing) release of openHAB is now installed on your system. Please test the correct behavior of your setup. You might need to adapt your configuration, if available. If you did changes to files below '/var/lib/openhab2' before, they were replaced but you can restore them from backup files next to the originals.
```
Comment (EliasGabrielsson) [inline functions/openhab.sh]: `The milestone release (testing ...`

**Comment by EliasGabrielsson** [comment]
Thanks for the work Markus 👌. I will try to get some time of and review your other pull request upcoming month.

**Comment by mstormi** [comment]
ok now ?

**Comment by EliasGabrielsson** [comment]
Well done!

**Pull request merged**